packages/studio-be/src/studio/nlu/nlu-client/index.ts

```diff
@@ -77,6 +77,15 @@ export class NLUClient {
     }
   }
 
+  public async downloadModelWeights(appId: string, modelId: string): Promise<Buffer> {
+    const downloadRes = await this._client.modelWeights.download(appId, modelId, { responseType: 'arraybuffer' })
+    if (downloadRes.status !== 'OK') {
+      throw new Error(`Download weights received status ${downloadRes.status}`)
```

Comment:
I don't know what would be a better error than the one suggested in the client's README. Any idea @franklevasseur ?

Reply:
this is good